package.json

```diff
@@ -18,17 +20,22 @@
   },
   "homepage": "https://github.com/feckwame/chris-chewy-description#readme",
   "dependencies": {
+    "aws-sdk": "^2.729.0",
     "bootstrap": "^4.5.0",
     "cors": "^2.8.5",
     "css-loader": "^3.6.0",
     "express": "^4.17.1",
+    "html-webpack-plugin": "^4.3.0",
+    "jquery": "^3.5.1",
     "mongoose": "^4.11.6",
     "multer": "^1.4.2",
     "node-fetch": "^2.6.0",
+    "body-parser": "^1.19.0",
     "react": "^16.13.1",
     "react-bootstrap": "^1.3.0",
     "react-dom": "^16.13.1",
-    "react-router-dom": "^5.2.0"
+    "react-router-dom": "^5.2.0",
+    "webpack-merge": "^5.1.1"
```

Comment:
not sure this needs to be a dependency over dev-dependency. same for some other loaders and plugins